python/cugraph/cugraph/tests/test_node2vec.py

```diff
@@ -172,14 +96,202 @@ def test_node2vec_invalid(
+        # in all of the paths are valid and are accurate
+        for i in range(k):
+            path_at_end, j = False, 0
+            weight_idx = 0
+            while not path_at_end:
+                src_idx = i * max_depth + j
+                dst_idx = i * max_depth + j + 1
+                if directed:
+                    invalid_src = blanks[src_idx] or (src_idx >= num_verts)
+                    invalid_dst = blanks[dst_idx] or (dst_idx >= num_verts)
+                    if invalid_src or invalid_dst:
+                        break
+                weight = edge_weights[weight_idx]
+                u = vertex_paths[src_idx]
+                v = vertex_paths[dst_idx]
+                # Walk not found in edgelist
+                edge_found = G.has_edge(u, v)
+                if not edge_found:
+                    raise ValueError("Edge {},{} not found".format(u, v))
+                # Corresponding weight to edge is not correct
+                expr = "(src == {} and dst == {})".format(u, v)
+                edge_query = G.edgelist.edgelist_df.query(expr)
+                if edge_query.empty:
+                    raise ValueError("edge_query didn't find:({},{}),{}".
+                                     format(u, v, num_verts))
+                else:
+                    if edge_query["weights"].values[0] != weight:
+                        raise ValueError("edge_query weight incorrect")
+
+                # Only increment if the current indices are valid
+                j += 1
+                weight_idx += 1
+                if j >= max_depth - 1:
+                    path_at_end = True
+            # Check that path sizes matches up correctly with paths
+            # FIXME: For some unknown reason, the start of each path is
+            # 2x the intended index when the path is padded. So if each
+            # path has 4 walks / 5 vertices visited, path 2 starts at index
+            # 10 instead of index 5, the below commented test works
+            # if i * max_depth * 2 < vertex_paths.size and \
+            #   vertex_paths[i * max_depth * 2] != seeds[i]:
+            # if vertex_paths[i * max_depth] != seeds[i]:
+            #    raise ValueError("vertex_path start did not match seed \
+            #                     vertex:{}".format(vertex_paths.values))
+
+
+@pytest.mark.parametrize(*_get_param_args("graph_file", [LINE]))
+@pytest.mark.parametrize(*_get_param_args("renumber", [True, False]))
+def test_node2vec_renumber_cudf(
+    graph_file,
+    renumber
+):
+    cu_M = cudf.read_csv(graph_file, delimiter=' ',
+                         dtype=['int32', 'int32', 'float32'], header=None)
+    G = cugraph.Graph(directed=True)
+    G.from_cudf_edgelist(cu_M, source="0", destination="1", edge_attr="2",
+                         renumber=renumber)
+
+    # if graph_file == KARATE:
+    #     start_vertices = [12, 28, 20, 23]
+    # else:
+    start_vertices = [8, 0, 7, 1, 6, 2]
+    num_seeds = 4
```

Comment:
Why is `len(start_vertices) = 6` and `num_seeds = 4`? Did you forget to update `num_seeds` after you commented part of your code?

Reply:
It was that. Updating the value of num_seeds did not affect the correctness of the test